spyder/widgets/tests/test_collectioneditor.py

```diff
@@ -333,6 +334,9 @@ def test_shows_dataframeeditor_when_editing_index(monkeypatch):
 
 
 def test_sort_numpy_numeric_collectionsmodel():
+    if parse(numpy.__version__) >= parse("2.0.0"):
+        numpy.set_printoptions(legacy="1.25")
+
```

Comment:
I believe this change is not "tempoary" and might hide failures in other tests.

Perhaps it is better to consider the context manager version of this function:

numpy.printoptions
https://numpy.org/doc/stable/reference/generated/numpy.printoptions.html#numpy.printoptions

Reply:
> I believe this change is not "tempoary" and might hide failures in other tests.

Ok, I'll fix that by actually updating the test to work with both Numpy 1 and 2 (I was a bit lazy and didn't want to do it for now).

Reply:
awesome, truthfully you are all quite thorough testing the front end as well as the data and results.

Reply:
Thanks @hmaarrfk!